pkg/process/checks/chunking.go

```diff
@@ -7,95 +7,9 @@ package checks
 
 import (
 	model "github.com/DataDog/agent-payload/v5/process"
+	"github.com/DataDog/datadog-agent/pkg/process/util"
 )
 
-// payloadList is an abstract list of payloads subject to chunking
-type payloadList interface {
-	// Len returns the length of the list
```

Comment:
Move this part to the third party package `pkg/process/util/chunking.go` to avoid cycle import.